tests/test_scrubbers.py

```diff
@@ -113,3 +118,71 @@ def test_faker_scrubber_run_twice(self):
 
         self.assertNotEqual(data.company, 'Foo')
         self.assertNotEqual(data.company, '')
+
+    @override_settings(DEBUG=True)
```

Comment:
we can probably leave the `DEBUG=True` overrides out, right?

Reply:
ah sorry, I missed the fact that you weren't using `with self.settings` like the other tests. All good.